app/user.go

```diff
@@ -999,6 +1009,18 @@ func (a *App) UpdateActive(user *model.User, active bool) (*model.User, *model.A
 		}
 	}
 
+	directChannels, e := a.Srv().Store.Channel().GetDirectChannelsForUser(user.Id)
+	if e != nil {
+		return nil, e
+	}
+
+	for _, channel := range directChannels {
+		if active {
+			updateTime = 0
+		}
```

Comment:
Any reason this is inside the loop? The `active` variable doesn't seem to change.

Reply:
Done